test/role_test.js

```diff
@@ -138,6 +139,52 @@ helper.secrets.mockSuite(helper.suiteName(__filename), ['app', 'azure'], functio
     assert(roles.some(role => role.roleId === 'thing-id:' + clientId));
   });
 
+  test('listRoleIds', async () => {
+    await helper.Roles.modify((roles) => roles.splice(0));
```

Comment:
What are we trying to do here? `roles.splice(0)` destroys the old array and replaces it with a copy. Perhaps add a comment above to make it clearer :)

Reply:
This is to clear the roles created by the other tests. I'll add a comment explaining it :)